### DISCUSSION TITLE
SpeechSynthesisUtterance issue

### DISCUSSION THREAD

**Discussion by ArpitQuest** [mdn/mdn-community#457] [Browser compatibility data]
I'm facing an issue in SpeechSynthesisUtterance when I use this in Chrome browser on Android devices. The voice gets automatically off, and only some portion of the sentence is played. I'm using this in my React project.
Thanks

**Comment by pepelsbey** [comment]
Thank you for starting the discussion. I’m closing this now due to inactivity. I’d also suggest joining [MDN Discord](https://developer.mozilla.org/discord) if you still have any questions you’d like to discuss with the community.
